### ISSUE TITLE
After converting from .pt to .torchscript, bounding box is off

### ISSUE THREAD

**Issue by willsonyee** [ultralytics/yolov5#6947]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

I am playing around with YOLOv5 and exporting to different formats (.pt & .torchscript). But I found that after I exported the pre-trained models using this command: 
`!python "/content/drive/MyDrive/yolov5/export.py" --weights "/content/drive/MyDrive/yolov5/yolov5s.pt" --include torchscript`

The .torchscript model's bounding is off, please advise me on why this is happening and how to resolve it. Thank you.

**.pt model:**
![zidane1](https://user-images.githubusercontent.com/98082493/157879125-eec4812e-d512-48d8-965b-8d8d4f00320b.jpeg)

**.torchscript model:**
![zidane2](https://user-images.githubusercontent.com/98082493/157879135-2dd699c4-6cbe-4b17-9771-25e55e50ccb9.jpeg)

**Comment by glenn-jocher** [comment]
@willsonyee 👋 hi, thanks for letting us know about this possible problem with YOLOv5 🚀. Torchscript export and inference works well for me, I'm unable to reproduce any box issues.

<img width="865" alt="Screenshot 2022-03-11 at 14 56 32" src="https://user-images.githubusercontent.com/26833433/157881457-8599a014-8634-4eb4-a6a2-754c0df40858.png">

We've created a few short guidelines below to help users provide what we need in order to get started investigating a possible problem.

## How to create a Minimal, Reproducible Example

When asking a question, people will be better able to provide help if you provide **code** that they can easily understand and use to **reproduce** the problem. This is referred to by community members as creating a [minimum reproducible example](https://docs.ultralytics.com/help/minimum_reproducible_example/). Your code that reproduces the problem should be:

- ✅ **Minimal** – Use as little code as possible to produce the problem
- ✅ **Complete** – Provide all parts someone else needs to reproduce the problem
- ✅ **Reproducible** – Test the code you're about to provide to make sure it reproduces the problem

For [Ultralytics](https://ultralytics.com/) to provide assistance your code should also be:

- ✅ **Current** – Verify that your code is up-to-date with GitHub [master](https://github.com/ultralytics/yolov5/tree/master), and if necessary `git pull` or `git clone` a new copy to ensure your problem has not already been solved in [master](https://github.com/ultralytics/yolov5/tree/master).
- ✅ **Unmodified** – Your problem must be reproducible using official YOLOv5 code without changes. [Ultralytics](https://ultralytics.com/) does not provide support for custom code ⚠️.

If you believe your problem meets all the above criteria, please close this issue and raise a new one using the 🐛 **Bug Report** [template](https://github.com/ultralytics/yolov5/issues/new/choose) with a [minimum reproducible example](https://docs.ultralytics.com/help/minimum_reproducible_example/) to help us better understand and diagnose your problem. 

Thank you! 😃

**Comment by willsonyee** [comment]
@glenn-jocher thanks for your prompt response.

I am running the code like this:
```
import torch
model = torch.hub.load('/content/drive/MyDrive/yolov5', 'custom', path='/content/drive/MyDrive/yolov5/yolov5s.torchscript', source='local')
result = model("/content/drive/MyDrive/yolov5/data/images/zidane.jpg")
result.save() 
```

The reason I am doing it like this because I am trying to load the model from local and inference without using the detect.py. So I can later on integrate the .torchscript model into my swift IOS app.

Could you advise me why by running the code I attached will cause the bounding box off ?

**Comment by glenn-jocher** [comment]
@willsonyee ah yes you are right there is a bug. I'm able to reproduce with this code:
```python
!python export.py --weights yolov5s.pt --include torchscript
model = torch.hub.load('ultralytics/yolov5', 'custom', path='yolov5s.torchscript')
result = model("data/images/zidane.jpg")
result.save()
```
<img width="980" alt="Screenshot 2022-03-11 at 15 21 14" src="https://user-images.githubusercontent.com/26833433/157885509-b5fb73cb-7782-4e70-8d2a-6dab7511a366.png">

I'll take a look!

**Comment by glenn-jocher** [comment]
@willsonyee good news 😃! Your original issue may now be fixed ✅ in PR #6949. 

![image](https://user-images.githubusercontent.com/26833433/157895360-1c156c47-8467-4ff0-81c1-a68638e9e1d7.png)

To receive this update:
- **[Git](https://github.com/ultralytics/yolov5)** – `git pull` from within your `yolov5/` directory or `git clone https://github.com/ultralytics/yolov5` again
- **[PyTorch Hub](https://pytorch.org/hub/ultralytics_yolov5/)** – Force-reload `model = torch.hub.load('ultralytics/yolov5', 'yolov5s', force_reload=True)`
- **[Notebooks](https://github.com/ultralytics/yolov5/blob/master/tutorial.ipynb)** – View updated notebooks  <a href="https://colab.research.google.com/github/ultralytics/yolov5/blob/master/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/ultralytics/yolov5"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **[Docker](https://hub.docker.com/r/ultralytics/yolov5)** – `sudo docker pull ultralytics/yolov5:latest` to update your image <a href="https://hub.docker.com/r/ultralytics/yolov5"><img src="https://img.shields.io/docker/pulls/ultralytics/yolov5?logo=docker" alt="Docker Pulls"></a>

Thank you for spotting this issue and informing us of the problem. Please let us know if this update resolves the issue for you, and feel free to inform us of any other **issues** you discover or **feature requests** that come to mind. Happy trainings with YOLOv5 🚀!
